framework/validators/AtLeastValidator.php

```diff
@@ -72,33 +63,42 @@ class AtLeastValidator extends Validator
     public function init()
     {
         parent::init();
-        if (empty($this->with)) {
-            throw new InvalidConfigException('Param "with" can not be null');
+        if ($this->isEmpty($this->alternativeAttributes)) {
+            $this->alternativeAttributes = (array)$this->attributes;
+        } else {
+            $this->alternativeAttributes = array_merge((array)$this->attributes, (array)$this->alternativeAttributes);
         }
+        $this->errorAttributes = $this->isEmpty($this->errorAttributes) ? (array)$this->alternativeAttributes : (array)$this->errorAttributes;
```

Comment:
according to $errorAtrributes definition, this line should be an array_merge between attributes and alternativeAttributes when errorAttributes is empty.

Reply:
[`alternativeAttributes`](https://github.com/developeruz/yii2/blob/e30ede98ce1c71c13d31f6acdb79bab33362e968/framework/validators/AtLeastValidator.php#L69) is contain array_merge between `attributes` and `alternativeAttributes` (couple lines above)

Reply:
👍 missed that. my bad.